### DISCUSSION TITLE
App Migration Tool - Prototype Testing

### DISCUSSION THREAD

**Discussion by 0x-allan** [akash-network/community#651] [Ideas]
idk if this is the right place for this, but i'm working on a grad school project focused on service design. i chose to focus on Akash and one of the ideas i'm prototyping is an "App Migration Tool" that lets devs easily port over their apps from traditional cloud platforms to Akash.

if you're willing to spend a few minutes reviewing the prototype and leaving any feedback you may have, i would really appreciate it! The (purely visual) prototype is accessible via this Miro link: https://miro.com/app/board/uXjVKsjexs8=/?share_link_id=831075494007

sent this message in discord originally and was pointed in this general direction - lmk if this would be better shared somewhere else!

discord handle is: `allan.g`

**Comment by dominikusbrian** [comment]
Just added a series of comments to the Miro board. This is an interesting project that will help in growing user channel growth pipeline for Akash Network. 

Would be nice to also see more on the technical aspects on how this going to be implemented. Be sure to check this proposal that has been approved and currently being worked on https://github.com/orgs/akash-network/discussions/606 by many of the Akash Network long-time contributors and developers. A huge part of the Akash Console V2 upgrade will have the automated deployment from GitHub/GitLab/BitBucket too.

**Reply by 0x-allan** [reply]
extremely helpful - thank you @dominikusbrian!

i'll be sure to review the Console V2 project also.

**Reply by mx8820M** [reply]
Hi @dominikusbrian Cannot find proposal 606 anymore though. Any news on the Console V2 yet actually?

**Reply by dominikusbrian** [reply]
Yes Console V2 have been progressing much. You can check the regular meeting update here : https://github.com/akash-network/community/tree/main/sig-clients/meetings
Feel free to also tune in to the ongoing discussion and progress update through the Akash Discord Server sig-client. 
In replacement of the discussion 606, you can check on the corresponding proposal here : https://www.mintscan.io/akash/proposals/258
